### PR TITLE
WIP: Editor video block with vuejs comp linked with consent dialog

### PR DESCRIPTION
**Pull request by Katrienvh**
## Type

- Enhancement

## Pull request description

1. vueJS components to show/hide the embed video based on the consent dialog visitor choice.

Component props: 
- Id: required
- notAllowedMessage: optional (translation)
- allowButtonText: optional (translation)
- level: optional (consent dialog level)

2. Fix editor js image block

TODO:
- Implement comp with editorjs video block

Example: 

image

image

image

### CODE REVIEW

File: src/Frontend/Core/Js/Vue-components/VEmbed.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        type: String,
+        required: true
+      },
+      notAllowedMessage: String,
+      allowButtonText: String,
+      level: String
+    },
+    data () {
+      return {
+        url: 'https://www.youtube.com/embed/',
```
Comment (tijsverkoyen) [inline src/Frontend/Core/Js/Vue-components/VEmbed.vue:36]: Maybe use https://www.youtube-nocookie.com/embed/ per default? See https://support.google.com/youtube/answer/171780?hl=en-GB#zippy=%2Cturn-on-privacy-enhanced-mode

File: src/Frontend/Themes/Fork/Core/Layout/Templates/PrivacyConsentDialog.html.twig [hunk 2]
```diff
@@ -41,7 +41,7 @@
                     data-value="{{ level }}"
                     class="custom-control-input"
                   >
-                  <label id="privacyConsentLevel{{ level|ucfirst }}Checkbox" class="custom-control-label">
+                  <label for="privacyConsentLevel{{ level|ucfirst }}Checkbox" id="privacyConsentLevel{{ level|ucfirst }}Checkbox" class="custom-control-label">
```
Comment (tijsverkoyen) [inline src/Frontend/Themes/Fork/Core/Layout/Templates/PrivacyConsentDialog.html.twig:44]: 👍

**Review by tijsverkoyen — COMMENTED** [review]
Can you also add a preview to this PR please.

**Comment by Katrienvh** [comment]
First going to merge it in [add-more-editor-blocks](https://github.com/forkcms/forkcms/tree/add-more-editor-blocks). Later that will be merge in restructure-core.

**Pull request closed without merging**